schema/statement.json

```
@@ -11,19 +11,19 @@
   "$defs": {
     "Statement": {
         "title": "Statement",
-        "description": "A statement about a person, entity or relationship, made at a particular point in time.",
+        "description": "A claim about a person, entity or relationship, made at a particular point in time.",
         "type": "object",
         "properties": {
           "statementId": {
             "title": "Statement Identifier",
-            "description": "A persistent globally unique identifier for this statement.",
+            "description": "A persistent globally unique identifier for this Statement. Length MUST be 32 - 64 characters (inclusive).",
             "type": "string",
             "minLength": 32,
             "maxLength": 64
           },
           "statementDate": {
-            "title": "Statement date",
-            "description": "The date (optionally including time) on which this statement was declared by the source, in YYYY-MM-DDTHH:MM:SS or YYYY-MM-DD format.",
+            "title": "Statement Date",
+            "description": "The date (optionally including time) on which this statement was declared by the source, in full-date (YYYY-MM-DD) or date-time (e.g. YYYY-MM-DDTHH:MM:SSZ) format. See the IETF RFC3339 standard, section 5.6.",
```

```suggestion
            "description": "The date (optionally including time) on which this Statement was declared by the source, in full-date (YYYY-MM-DD) or date-time (e.g. YYYY-MM-DDTHH:MM:SSZ) format. See the IETF RFC3339 standard, section 5.6.",
```

or if statement is being capitalised then this should be?

---

I think we're referring to the concept (not data element) here. Leaving for now.